ios/Classes/PTFlutterDocumentController.m

```diff
@@ -797,6 +830,27 @@ - (BOOL)toolManager:(PTToolManager *)toolManager shouldHandleLinkAnnotation:(PTA
     return YES;
 }
 
+#pragma mark - Notification
+
+- (void)undoManagerSentNotification:(NSNotification *) notification
+{
+    if (self.isAnnotationManagerEnabled && self.userId) {
+        NSString* xfdf = [self.pdfViewCtrl.externalAnnotManager GetLastXFDF];
+        NSError* error;
+        
+        [self.toolManager.annotationManager updateAnnotationsWithXFDFCommand:xfdf error:&error];
```

Comment:
I'm a bit surprised this is necessary. Please confirm that it is @dluco ?

Reply:
@JamesTron Sorry, I got mixed up as to where `updateAnnotationsWithXFDF...` would be called - you're correct that it isn't necessary here.
```suggestion
```